objects_tdd/lib/cash_register.rb

```diff
@@ -0,0 +1,24 @@
+# is a cashRegister
+class CashRegister
+  attr_reader :total
+  def initialize
+    @total = 0
+  end
+
+  def add(number)
+    @total += number
+  end
+
+  def subtract(number)
+    @total -= number
+  end
+
+  def pay(number)
+    @total -= number
+    if @total < 0
```

Comment:
This is throwing a guard error because there is no `else` statement to compare it to. So what would happen is total was greater than 0? Just need to rework your code a little...